static/app/views/issueList/utils.tsx

```diff
@@ -217,3 +217,32 @@ export const FOR_REVIEW_QUERIES: string[] = [Query.FOR_REVIEW];
 
 export const SAVED_SEARCHES_SIDEBAR_OPEN_LOCALSTORAGE_KEY =
   'issue-stream-saved-searches-sidebar-open';
+
+export enum IssueGroup {
+  ALL = 'all',
+  ERRORS_OUTAGES = 'errors_outages',
+  TRENDS = 'trends',
+  CRAFTSMANSHIP = 'craftsmanship',
+  SECURITY = 'security',
+}
+
+const IssueGroupFilter: Record<IssueGroup, string> = {
+  [IssueGroup.ALL]: '',
+  [IssueGroup.ERRORS_OUTAGES]: 'issue.category:[error,cron,uptime]',
+  [IssueGroup.TRENDS]:
+    'issue.type:[profile_function_regression,performance_p95_endpoint_regression,performance_n_plus_one_db_queries]',
+  [IssueGroup.CRAFTSMANSHIP]:
+    'issue.category:replay issue.type:[performance_n_plus_one_db_queries,performance_n_plus_one_api_calls,performance_consecutive_db_queries,performance_render_blocking_asset_span,performance_uncompressed_assets,profile_file_io_main_thread,profile_image_decode_main_thread,profile_json_decode_main_thread,profile_regex_main_thread]',
+  [IssueGroup.SECURITY]: 'event.type:[nel,csp]',
+};
```

Comment:
Worth discussing if these should be surfaced in the URL as raw search params under `query` or handled as a new construct (with a proper `group` param, maybe?). If these are intended to be hidden from the user search bar, a new construct would provide a more seamless user experience.

Reply:
After discussing in person, we decided to consider this version as a prototype of the new experience that might possibly roll into custom view updates in the future.